expediagroup-sdk-rest/build.gradle

```diff
@@ -41,7 +41,8 @@ kover {
         filters {
             excludes {
                 packages(
-                        "com.expediagroup.sdk.rest.trait",
+                    "com.expediagroup.sdk.rest.trait",
+                    "com.expediagroup.sdk.rest.exception",
```

Comment:
It does not need to be excluded. We can add any test to any scenario that triggers such an exception, and it should be covered. I will approve this anyway, but please keep this in mind when expanding our tests.